main.css

```diff
@@ -0,0 +1,76 @@
+a {
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  height: auto;
+  background-color: black;
+  color: white;
+}
+
+.header-left {
+  font-style: italic;
+  padding-left: 5px;
+}
+
+header li {
+  display: inline-block;
+  padding: 5px;
+}
+
+#main {
+  text-align: center;
+  font-size: 20px;
+  background-color: #eee;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+#img-div {
+  background-color: white;
+  padding: 5px 0;
+}
+
+#img-caption {
+  font-size: 15px;
+  margin-bottom: 30px;
+}
+
+img {
+  border-radius: 2px;
+  width: 80%;
+  height: 80%;
+  margin-bottom: 20px;
+}
+
+.timeline-div {
+  padding-top: 50px;
+  width: 50%;
+  margin: 0 auto;
+}
+
+.timeline-div li {
+  text-align: left;
```

Comment:
I think you used text align: left too many times in different place.You can make a class just for that, so you don't clutter your CSS file.

Reply:
This is a great comment! 👍 
It is written in a polite tone and also explains the reason for the requested change.